### PR TITLE
Rebase main

### PR DESCRIPTION
**Pull request by duckatgit**

### CODE REVIEW

File: lib/screens/match/page/match_page.dart [hunk 1]
```diff
@@ -31,9 +34,11 @@ class MatchPage extends StatelessWidget {
         BlocProvider.value(value: getIt<HomeStorageBloc>()),
         BlocProvider(create: (_) => MatchBloc()..add(OnInitUserID())),
         BlocProvider(
-            create: (_) => user.ProfileBloc()..add(user.OnInitUserID())),
+            create: (_) => userProfile.ProfileBloc()..add(userProfile.OnInitUserID())),
         BlocProvider(create: (_) => getIt<FetchFeedBloc>()),
         BlocProvider(create: (_) => getIt<LikeUserBloc>()),
+        BlocProvider(create: (_) => getIt<FetchUserBloc>()),
+        BlocProvider(create: (_) => getIt<FetchVerifyRequestsBloc>()),
```
Comment (vikalekhlitslink) [inline lib/screens/match/page/match_page.dart:41]: Why do we need the 'FetchVerifyRequestsBloc' here?

File: lib/data/text_constants.dart [hunk 2]
```diff
@@ -396,5 +396,7 @@ class TextConstants {
   // -> Premium dialog
   static const String outOfSuperLikes = "Out of Super Likes!";
   static const String doNotLoseDiamonds = "Don't lose diamonds, Refill now!";
+  static const String superLikes = "Super Likes";
+  static const String dialogBuyButton = "Buy Super Likes for \$7.99";
```
Comment (vikalekhlitslink) [inline lib/data/text_constants.dart:400]: Why 7.99 when we have 3 prices that will be changing?

File: lib/screens/premium_screen/widgets/premium_purchase_dialog.dart [hunk 3]
```diff
@@ -315,35 +146,33 @@ Widget _buttonContainer({
...
+      padding: const EdgeInsets.all(4),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         crossAxisAlignment: CrossAxisAlignment.center,
         children: [
           FittedBox(
             child: Text(
               '${subscriptionPlan.superLikeCount}',
-              style: getTextTheme(context).headlineMedium,
+              style: getTextTheme(context).bodyMedium,
```
Comment (vikalekhlitslink) [inline lib/screens/premium_screen/widgets/premium_purchase_dialog.dart:161]: change text style

File: lib/screens/match/page/match_page.dart (hunk 1 above)
Comment (duckatgit) [reply lib/screens/match/page/match_page.dart:41]: I removed the 'FetchVerifyRequestsBloc' from here.

**Pull request merged**